pulp_ansible/tests/functional/api/collection/v3/test_serializers.py

```diff
@@ -32,61 +30,62 @@ def setUpClass(cls):
         cls.collections_versions_api = ContentCollectionVersionsApi(cls.client)
         cls.collections_versions_v3api = PulpAnsibleGalaxyApiV3VersionsApi(cls.client)
 
-    def setUp(self):
-        """Create distribution for V3."""
-        repo = self.repo_api.create(gen_repo())
-        self.addCleanup(self.repo_api.delete, repo.pulp_href)
-        self.repo_href = repo.pulp_href
-        requirements = "collections:\n  - name: pulp.pulp_installer"
-
-        body = gen_ansible_remote(url="https://galaxy.ansible.com", requirements_file=requirements)
-        remote = self.remote_collection_api.create(body)
-        self.addCleanup(self.remote_collection_api.delete, remote.pulp_href)
-
-        # Sync the repository.
-        repository_sync_data = RepositorySyncURL(remote=remote.pulp_href)
-        sync_response = self.repo_api.sync(repo.pulp_href, repository_sync_data)
-        monitor_task(sync_response.task)
-        repo = self.repo_api.read(repo.pulp_href)
-
-        # Create a distribution.
-        body = gen_distribution()
-        body["repository"] = repo.pulp_href
-        distribution_create = self.distributions_api.create(body)
-        created_resources = monitor_task(distribution_create.task).created_resources
-        distribution = self.distributions_api.read(created_resources[0])
-        self.base_path = distribution.base_path
-        self.addCleanup(self.distributions_api.delete, distribution.pulp_href)
-
     @unittest.skip("FIXME: Re-enable later")
```

Comment:
This test is being skipped do we still want this skipped?

Reply:
Oh is this what is enabled in the other PR actually?

Reply:
yes, this is re-enabled on `update_at/uqery performance` PR